store/sqlstore/store.go

```diff
@@ -920,7 +920,11 @@ func (ss *SqlStore) AlterColumnTypeIfExists(tableName string, columnName string,
 	}
 
 	if err != nil {
-		mlog.Critical("Failed to alter column type", mlog.Err(err))
+		msg := "Failed to alter column type."
+		if mySqlColType == "JSON" && postgresColType == "jsonb" {
+			msg += " It is likely you might have invalid JSON values in the column. Please fix the values manually and run the migration again."
```

Comment:
0/5 on a better error message here. Let me know if you want something changed.

Reply:
I'd remove the "might" since it's already a conditional but other than that sounds good to me.

Reply:
```suggestion
			msg += " It is likely you have invalid JSON values in the column. Please fix the values manually and run the migration again."
```